test/attr-for-selected.html

```diff
@@ -210,18 +210,26 @@
       });
 
       test('selection is updated after fallback is set', function() {
-        selector.fallbackSolution = undefined;
+        selector.fallbackSelection= undefined;
         selector.select('non-existing-value');
```

Comment:
So long as we're cleaning up these tests.. :) I think this assertion needs to be on the next line: `assert.equal(selector.selectedItem, items[0]);`.